### PR TITLE
Optimize workflow - typedoc and filename

### PR DESCRIPTION
**Pull request by VisualSJ**
Re: #

### Changelog

* remove typedoc dependencies
* typo

### CODE REVIEW

**Review by minggo — APPROVED** [review]

File: scripts/generate-doc-info.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+async function exec() {
+    await bash('npm', ['install', 'typedoc@0.22.18', '--no-save'], {
+        cwd: PATHS.engine,
+    });
+
+    await bash('npm', ['install', '--only=production'], {
+        cwd: PATHS.plugin,
+    });
+    
+    await bash('npm', ['install', './scripts/typedoc-plugin', '--no-save'], {
```
Comment (dogodo-cc) [inline scripts/generate-doc-info.js:38]: 'typedoc@0.22.18' &  './scripts/typedoc-plugin'
Can be combined into a single line， like
['install', 'typedoc@0.22.18', './scripts/typedoc-plugin'，'--no-save']

File: package.json [hunk 2]
```diff
@@ -13,23 +13,23 @@
...
     "fetch-ral": "node ./scripts/fetch-ral.js",
     "bundle-adapter": "node ./scripts/bundle-adapter.js",
-    "build-const": "node ./scripts/build-consts.js",
+    "build-const": "node ./scripts/build-const.js",
     "clear": "node ./scripts/clear-cache.js",
     "clear-platform": "node ./native/utils/clear-platform.js && node ./scripts/clear-file.js",
     "update:native-external": "node ./native/utils/update-external.js",
     "test": "gulp test",
-    "install:typedoc-plugin": "cd ./scripts/typedoc-plugin && npm install --only=production",
+    "generate-doc-info": "node ./scripts/generate-doc-info",
```
Comment (dogodo-cc) [inline package.json:32]: need update the api repo`s README.md

File: scripts/generate-doc-info.js (hunk 1 above)
Comment (VisualSJ) [reply scripts/generate-doc-info.js:38]: But here we need to install the typedoc first, before compiling, and finally we can install this plugin

File: package.json (hunk 2 above)
Comment (VisualSJ) [reply package.json:32]: There is no explanation in readme

Comment (dogodo-cc) [reply package.json:32]: http://git.cocos.org:9080/G_Engine/cocos3d-docs-api/-/tree/v3.8.1 in this repo, we have a readme  show how to release a api doc.

**Review by dogodo-cc — APPROVED** [review]

**Pull request merged**